packages/docutils/lib/cli/command/validate.ts

```diff
@@ -76,25 +84,25 @@ const opts = Object.freeze({
   typescript: {
     default: true,
     description: 'Validate TypeScript environment',
-    group: NAME_GROUP_VALIDATE,
+    group: ValidateCommandGroup.Behavior,
     type: 'boolean',
   },
-}) satisfies Record<string, Options>;
+} as const satisfies Record<string, Options>;
 
 type ValidateOptions = InferredOptionTypes<typeof opts>;
 
 export default {
   command: 'validate',
   describe: 'Validate Environment',
-  builder: opts,
+  builder(yargs) {
+    return yargs.options(opts).check(async (argv) => {
+      if (!argv.python && !argv.typedoc && !argv.typescript && !argv.mkdocs) {
+        return 'No validation targets specified; one or more of --python, --typescript, --typedoc or --mkdocs must be provided';
+      }
+      return checkMissingPaths(opts, ValidateCommandGroup.Paths, argv);
```

Comment:
await?

Reply:
more about this: my rationale for not using it is that it does not matter what `checkMissingPaths` returns because the function consuming it is `async`; adding `await` here without a `try/catch` serves no practical purpose other than causing churn if it did happen to become a sync function (because keeping a useless `await` will be slow in a hot path and could be the source of subtle bugs).  

anyway, I'm going to leave this for now and I can make my case when we add `typescript-eslint` support.